rpc/common/admin/health/moc_common_health.proto

```diff
@@ -19,6 +19,7 @@ message HealthResponse {
 	string  Error = 3;
 	HealthState State = 4;
 	bool Rebooted = 5;
+	google.protobuf.BoolValue CertificateNeedsRenewal = 6;
```

Comment:
Abstract the hostRespose since certneedsrenewal is very specific to this use case